lib/travis/build/script/node_js.rb

```diff
@@ -41,7 +41,20 @@ def announce
 
         def install
           sh.if '-f package.json' do
-            sh.cmd "npm install #{config[:npm_args]}", retry: true, fold: 'install'
+            sh.if "-f yarn.lock" do
+              if version.to_i < 4
```

Comment:
This matches `stable`, `node`, and most notably `v4`, which will cause problems.

We should check the Node.js version at run time instead.